elastic/kibana
# [Uptime] Update breadcrumb generator for monitor -> overview.

## Summary

Resolves https://github.com/elastic/kibana/issues/49391.

Updates the breadcrumb generator in the Uptime app to more accurately reflect the destination of someone returning to the Uptime home page.

## Testing

0. Configure several Heartbeat monitors and navigate to the Uptime app in Kibana
1. Go to the _Overview_ page
2. Navigate to the _Monitor_ page for one of your monitors
3. Use the breadcrumb to navigate back. The path should reflect the home page.

Also be sure to test with base paths.

### Checklist

Use ~~strikethroughs~~ to remove checklist items you don't feel are applicable to this PR.


### For maintainers
